layouts/shortcodes/guides_list.html

```diff
@@ -8,7 +8,10 @@
 */}}
 
 {{ $icon := .Get "icon" }}
-{{ $guides := .Page.RegularPages }}
+{{ $category := .Get "category" }}
+  
+{{ $en_pages := where $.Site.AllPages "Lang" "en" }}
+{{ $guides := where $.Site.RegularPages ".Params.categories" "intersect" (slice $category) | lang.Merge (where $en_pages ".Params.categories" "intersect" (slice $category)) }}
```

Comment:
I am not sure whether or not there is a better way to do this...

I am listing English pages as fallbacks in the case of missing translations.